### PR TITLE
Add Plotting document autosummary

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

When trying to customize the axis display, I noticed that some of the APIs in the Plotting section, including the axis classes, were not being output to the documentation.  I have added classes to auto-summary that are worth adding to the documentation.

### Details

- None

### CODE REVIEW

**Comment by tkoyama010** [comment]
Please hold off merging until we merge #4486 and resolve the conflict with it.

**Review by banesullivan — APPROVED** [review]

**Pull request merged**
